fs.go

```diff
@@ -23,13 +24,13 @@ type FileSystem interface {
 	Reload(func(key, name string)) error
 
 	// Create should return a new File using a name generated from the passed key.
-	Create(key string) (File, error)
+	Create(key string) (stream.File, error)
```

Comment:
We might consider just embedding the stream.FileSystem interface in this interface.